fbw-common/src/systems/instruments/src/EFB/Ground/Pages/Fuel/A320_251N/A320Fuel.tsx

```diff
@@ -150,20 +150,22 @@ export const A320Fuel: React.FC<FuelProps> = ({
 
   const gsxRefuelCallable = () => gsxRefuelState === GsxServiceStates.CALLABLE;
 
-  const canRefuel = useCallback(
-    () => !(eng1Running || eng2Running || !isOnGround),
+  const onlyInstantRefuelAllowed = useCallback(
```

Comment:
Why does this need to use `useCallback`? Can we not use use this:
```typescript
const onlyInstantRefuelAllowed = eng1Running || eng2Running || !isOnGround;
```
I am wondering about the same thing for most of the other calls to `useCallback` in this file.